docs/reverse_proxy.md

```diff
@@ -98,10 +98,16 @@ example.com:8448 {
 
 ### Apache
 
+Enable `mod_proxy`, `mod_proxy_http`, `mod_proxy_balancer`, `mod_headers`
+and `mod_lbmethod_byrequests`.
+
 ```
 <VirtualHost *:443>
     SSLEngine on
-    ServerName matrix.example.com;
```

Comment:
Thanks for catching, `;` are indeed not a thing in Apache configs. And indeed https://httpd.apache.org/docs/2.4/mod/core.html#servername mentions to include a port for `ServerName` if possible:

> For optimal reliability and predictability, you should specify an explicit hostname and port using the ServerName directive.